front/components/ui/MainHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Flex, Text } from "@chakra-ui/react";
+import { ConnectButton } from "@rainbow-me/rainbowkit";
+import Link from "next/link";
+
+export const MainHeader = () => {
+  return (
+    <Box as="header">
+      <Flex alignItems="center" justifyContent="space-between">
+        <Link href="/">
+          <Text cursor="pointer">Home</Text>
```

Comment:
a nie wystarczy w taki sposób dodać linku?

```suggestion
          Home
```

Reply:
Nie, bo nie da się wystylować Link z Nexta. Jest też Link z Chakry, ale on jest mniej wygodny w użyciu według mnie, dlatego jest połączenie Link nextowego + wystowalowany Text chakry (przyda się jak już będziemy stylować menu:) )